react/RemoveButton.tsx

```diff
@@ -43,9 +43,7 @@ function RemoveButton(props: Props) {
 
   return (
     <div
-      className={`${handles.removeButtonContainer} ${opaque(
```

Comment:
can you explain why you are removing it?

Reply:
Sure! Because in a previous change request they told me that this change could break stores because the existing customization for removeButtonContainer's delete buttons on product gifts would be lost after merging. That's why I used the applyModifiers.

Reply:
I'm asking about removing the `removeButtonContainer` class. This may be a breaking change, right?

Reply:
I remove the removeButtonContainer because is necessary to create two different classes, one class for paid products (removeButtonContainer) and one class for free products to make the basket icon no longer visible if it is decided that the user should not be able to delete free products from the cart. I remove the removeButtonContainer becouse I needs to check if sellingPrice === 0 and to do this I use the applyModifiers. I hope I have clarified the doubts.

Reply:
Yeah, you are right, but do you agree that other stores that use this component and have styles for this modifier will break their UIs after this change?

Reply:
I think I understand what you mean, I've committed the changes